src/index.ts

```diff
@@ -43,7 +43,14 @@ export {RadixTrie, Leaf, eventToHotkeyString}
 export function install(element: HTMLElement, hotkey?: string): void {
   // Install the keydown handler if this is the first install
   if (Object.keys(hotkeyRadixTrie.children).length === 0) {
-    document.addEventListener('keydown', keyDownHandler)
+    const scopedElementId = element.getAttribute('data-hotkey-scope')
+    if (scopedElementId) {
+      const scopedElement = document.getElementById(scopedElementId)
+      if (!scopedElement) return
+      scopedElement.addEventListener('keydown', keyDownHandler)
```

Comment:
If we add event listeners to each scoped item here, we also want to remove them in `uninstall`. That's no big deal, but what we _might_ do instead is continue to use the global listener, and check if `event.currentTarget.id` matches `data-hotkey-scope`.